WebApp/autoreduce_webapp/reduction_viewer/views.py

```diff
@@ -103,7 +105,7 @@ def overview(_):
     Render the overview landing page (redirect from /index)
     Note: _ is replacing the passed in request parameter
     """
-    instruments = Instrument.objects.order_by('name')
+    instruments = VALID_INSTRUMENTS
```

Comment:
Should we assume that `VALID_INSTRUMENTS` is ordered alphabetically? Maybe we could do a sort here?
```suggestion
    instruments = VALID_INSTRUMENTS.sort()
```

or where `VALID_INSTRUMENTS` is defined:

```python
VALID_INSTRUMENTS = ['ENGINX', 'GEM', 'HRPD', 'MAPS', 'MARI', 'MUSR',
                     'OSIRIS', 'POLARIS', 'POLREF', 'WISH'].sort()
```

Reply:
Nice suggest Ricky, but I would prefer not to sort, because we may want to control the order manually that these are listed